resources/bundles/org.eclipse.core.filesystem/src/org/eclipse/core/internal/filesystem/local/LocalFileNativesManager.java

```diff
@@ -57,12 +58,11 @@ public static void reset() {
 	 */
 	public static boolean setUsingNative(boolean useNatives) {
 		boolean nativesAreUsed;
-		boolean isWindowsOS = Constants.OS_WIN32.equals(LocalFileSystem.getOS());
 
-		if (useNatives && !isWindowsOS && UnixFileNatives.isUsingNatives()) {
+		if (useNatives && UnixFileNatives.isUsingNatives()) {
```

Comment:
What about simplifying the nesting?

```java
if (useNatives) {
  HANLDER = new UnixFileHandler();
  nativesAreUsed = LocalFileNatives.isUsingNatives() || UnixFileNaties.isUsingNatives();
} else {
  nativesAreUsed = false;
 ...
}
```

Reply:
This will change the behavior, e.g. it creates the handler and is afterwards checking if it actually is sued, that does not make much sense to me, the whole purpose is to prevent the handler from being created and even try to load anything...